SConstruct

```diff
@@ -424,165 +429,190 @@ config_options = [
+        """The directory where the SUNDIALS header files are installed. This
            should be the directory that contains the "cvodes", "nvector", etc.
            subdirectories. Not needed if the headers are installed in a
-           standard location, e.g. /usr/include.""",
+           standard location, e.g., '/usr/include'.""",
         '', PathVariable.PathAccept),
     PathVariable(
         'sundials_libdir',
-        """The directory where the sundials static libraries are installed.
+        """The directory where the SUNDIALS static libraries are installed.
            Not needed if the libraries are installed in a standard location,
-           e.g. /usr/lib.""",
+           e.g., '/usr/lib'.""",
         '', PathVariable.PathAccept),
-    ('blas_lapack_libs',
-     """Cantera can use BLAS and LAPACK libraries available on your system if
-        you have optimized versions available (e.g. Intel MKL). Otherwise,
-        Cantera will use Eigen for linear algebra support. To use BLAS
-        and LAPACK, set blas_lapack_libs to the the list of libraries
-        that should be passed to the linker, separated by commas, e.g.
-        "lapack,blas" or "lapack,f77blas,cblas,atlas".""",
-     ''),
-    PathVariable('blas_lapack_dir',
-        """Directory containing the libraries specified by 'blas_lapack_libs'.""",
+    (
+        'blas_lapack_libs',
+        """Cantera can use BLAS and LAPACK libraries available on your system if
+           you have optimized versions available (e.g., Intel MKL). Otherwise,
+           Cantera will use Eigen for linear algebra support. To use BLAS
+           and LAPACK, set 'blas_lapack_libs' to the the list of libraries
+           that should be passed to the linker, separated by commas, e.g.,
+           "lapack,blas" or "lapack,f77blas,cblas,atlas".""",
+        ''),
+    PathVariable(
+        'blas_lapack_dir',
+        """Directory containing the libraries specified by 'blas_lapack_libs'. Not
+           needed if the libraries are installed in a standard location, e.g.
+           ``/usr/lib``.""",
         '', PathVariable.PathAccept),
     EnumVariable(
         'lapack_names',
         """Set depending on whether the procedure names in the specified
            libraries are lowercase or uppercase. If you don't know, run 'nm' on
-           the library file (e.g. 'nm libblas.a').""",
+           the library file (e.g., 'nm libblas.a').""",
         'lower', ('lower','upper')),
     BoolVariable(
-        'lapack_ftn_trailing_underscore', '', True),
+        'lapack_ftn_trailing_underscore',
+        """Controls whether the LAPACK functions have a trailing underscore
+           in the Fortran libraries.""",
+        True),
     BoolVariable(
-        'lapack_ftn_string_len_at_end', '', True),
+        'lapack_ftn_string_len_at_end',
+        """Controls whether the LAPACK functions have the string length
+           at the end in the Fortran libraries.""",
+        True),
```

Comment:
I meant that when I added the docs (there weren't any before) I'm not sure I described what this option does correctly, I just guessed based on how it's named. I think I'll update it slightly based on your comment. It seems like the other one is correct though?